css/ui/tooltips.css

```diff
@@ -399,8 +399,11 @@ html.playing .tooltip.hide-while-playing {
 
 .scrubber-handle .flipped-arrow {
     display: inline-block;
-    transform: rotate(180deg);
-    -webkit-transform: rotate(180deg);
+    transform: scaleX(-1.0);
+    -webkit-transform: scaleX(-1.0);
+    -moz-transform: scaleX(-1.0);
+    -ms-transform: scaleX(-1.0);
+    -o-transform: scaleX(-1.0);
```

Comment:
I'd like to suggest migrating to stylus + nib for doing CSS.  nib provides auto-prefixing and stylus could simplify our nested rules and provide nice things like variables for common colors and stuff.

http://learnboost.github.io/stylus/
http://nibstyl.us/docs/#transparent-mixins